blurb/blurb.py

```diff
@@ -888,6 +878,18 @@ def init_tmp_with_template():
     path = blurb.save_next()
     git_add_files.append(path)
     flush_git_add_files()
+
+    # For convenience, write a preformatted commit template. Enable template
```

Comment:
Again, a minor suggestion regarding your English here.  The first sentence is fine, for the second may I suggest "You can tell git to use this template with "git  config commit.template .git/blurb"."

It's not a big deal, but I don't know why you use `` to quote the command here.  Simple double-quotes are fine, and if you want something clearer I would indent and put a % to indicate a command prompt.

Also, based on the code, it looks like

1. this only works if the ~/.git/blurb directory exists, and
2. blurb doesn't create the directory for you automatically.

Does git create the directory for you?  If not, we should probably suggest that _too_ in the comment, or else people will follow the instructions only to discover that they don't work.